src/github/client/installation-id.ts

```diff
@@ -40,7 +40,8 @@ export class InstallationId {
 	}
 }
 
-export const getCloudInstallationId = (installationId: number, gitHubApiBaseUrl?: string | undefined): InstallationId => {
+export const getCloudInstallationId = (installationId: number, gitHubApiBaseUrl?: string | undefined, appId?: number): InstallationId => {
```

Comment:
rename the method? It is not `cloud` any more right? It includes the GHE stuff as well.

Reply:
Renamed to `getInstallationId`